src/satipc.c

```diff
@@ -496,6 +496,12 @@ void satip_close_device(adapter *ad)
 	satipc *sip = get_satip(ad->id);
 	if (!sip)
 		return;
+	if (ad->is_standby)
+	{
+		LOG("satip device %s:%d going to standby", sip->sip, sip->sport);
+		http_request(ad, NULL, "TEARDOWN");
```

Comment:
This is in no way different that the previous code: you are just seding the TEARDOWN. Please note that you are not resetting the session id which is not good.

Reply:
> This is in no way different that the previous code: you are just seding the TEARDOWN. Please note that you are not resetting the session id which is not good.

The objective is to just send the TEARDOWN. But, yes I don't have reseted the session id. I'll look for this. Any idea?